nevergrad/functions/gym/multigym.py

```diff
@@ -315,6 +315,12 @@ def get_env_names() -> tp.List[str]:
             try:
                 assert "Kelly" not in str(e.id)  # We should have another check than that.
                 assert "llvm" not in str(e.id)  # We should have another check than that.
+                assert "BulletEnv" not in str(e.id)  # We should have another check than that.
+                assert "Minitaur" not in str(e.id)  # We should have another check than that.
+                assert "Kuka" not in str(e.id)  # We should have another check than that.
+                assert (
+                    "RacecarZedBulletEnv-v0" != e.id
+                )  # This specific environment causes X11 error when using pybullet_envs.
```

Comment:
The error will be more explicit with a message than a comment
```suggestion
                assert (
                    "RacecarZedBulletEnv-v0" != e.id
                ), "This specific environment causes X11 error when using pybullet_envs."
```